packages/backend/src/accounting/psql/service.test.ts

```diff
@@ -767,4 +775,234 @@ describe('Psql Accounting Service', (): void => {
       )
     })
   })
+
+  describe('createTransfer', (): void => {
```

Comment:
I (basically) copied this from the tigerbeetle tests 🙃 

(to confirm feature parity)